### PR TITLE
[hotROD] Add OTEL instrumentation to customer svc

### PR DESCRIPTION
**Pull request by afzal442**
## Which problem is this PR solving?
Related to https://github.com/jaegertracing/jaeger/issues/3380
Part of https://github.com/jaegertracing/jaeger/issues/3381

## Short description of the changes
- This PR adds OTEL instrumentation to customer svc

### CODE REVIEW

File: examples/hotrod/services/customer/database.go [hunk 1]
```diff
@@ -73,15 +73,10 @@ func (d *database) Get(ctx context.Context, customerID string) (*Customer, error
 	d.logger.For(ctx).Info("Loading customer", zap.String("customer_id", customerID))
 
 	// simulate opentracing instrumentation of an SQL query
-	if span := opentracing.SpanFromContext(ctx); span != nil {
-		span := d.tracer.StartSpan("SQL SELECT", opentracing.ChildOf(span.Context()))
-		tags.SpanKindRPCClient.Set(span)
-		tags.PeerService.Set(span, "mysql")
```
Comment (yurishkuro) [inline examples/hotrod/services/customer/database.go:79]: Is there OTEL equivalent in the semantic conventions?

**Review by yurishkuro — COMMENTED** [review]
Please fill out proper pr description

File: examples/hotrod/services/customer/database.go [hunk 2]
```diff
@@ -19,8 +19,9 @@ import (
 	"context"
 	"errors"
 
-	"github.com/opentracing/opentracing-go"
-	tags "github.com/opentracing/opentracing-go/ext"
+	"go.opentelemetry.io/otel/attribute"
+	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
```
Comment (yurishkuro) [inline examples/hotrod/services/customer/database.go:23]: is that the latest version?

File: examples/hotrod/services/customer/database.go [hunk 3]
```diff
@@ -73,15 +74,10 @@ func (d *database) Get(ctx context.Context, customerID string) (*Customer, error
...
-		tags.SpanKindRPCClient.Set(span)
-		tags.PeerService.Set(span, "mysql")
-		// #nosec
-		span.SetTag("sql.query", "SELECT * FROM customer WHERE customer_id="+customerID)
-		defer span.Finish()
-		ctx = opentracing.ContextWithSpan(ctx, span)
-	}
+	ctx, span := d.tracer.Start(ctx, "SQL SELECT", trace.WithSpanKind(trace.SpanKindClient))
+	// #nosec
+	span.SetAttributes(semconv.RPCSystemKey.String("mysql"), attribute.Key("sql.query").String("SELECT * FROM customer WHERE customer_id=" + customerID))
```
Comment (yurishkuro) [inline examples/hotrod/services/customer/database.go:79]: RPCSystem: A string identifying the remoting system (such as gRPC or java_rmi) - does not match

Comment (yurishkuro) [inline examples/hotrod/services/customer/database.go:79]:
```suggestion
	span.SetAttributes(semconv.PeerServiceKey.String("mysql"), attribute.Key("sql.query").String("SELECT * FROM customer WHERE customer_id=" + customerID))
```

File: examples/hotrod/services/customer/database.go (hunk 2 above)
Comment (yurishkuro) [reply examples/hotrod/services/customer/database.go:23]: I see v1.20.0 -- https://pkg.go.dev/go.opentelemetry.io/otel@v1.16.0/semconv/

File: examples/hotrod/services/customer/database.go (hunk 3 above)
Comment (afzal442) [reply examples/hotrod/services/customer/database.go:79]: Ah, sorry. my bad again. I wasn't sure if `PeerServiceKey` exists.

**Review by yurishkuro — APPROVED** [review]

**Pull request merged**